docs/quick_start.md

```diff
@@ -0,0 +1,127 @@
+
+
+# Quick Start
```

Comment:
@angrybayblade is this a merge error? You didn't want to make any docs changes afaik